vars/tmsUpload.groovy

```diff
@@ -92,12 +92,13 @@ void call(Map parameters = [:]) {
             .withMandatoryProperty('credentialsId')
             .use()
 
+        def namedUser = jenkinsUtils.getJobStartedByUserId()
+
         if (config.useGoStep == true) {
             List credentials = [
                 [type: 'token', id: 'credentialsId', env: ['PIPER_tmsServiceKey']]
             ]
 
-            def namedUser = jenkinsUtils.getJobStartedByUserId()
             if (namedUser) {
                 parameters.namedUser = namedUser
             }
```

Comment:
Is it a problem for the golang implementation if namedUser is not set? Can it be set through config with the golang implementation?

Reply:
The parameter has a default value: https://github.com/SAP/jenkins-library/blob/549b32c67505bb512e902dbe263715df705a284a/resources/metadata/tmsUpload.yaml#L49
It can be set through config.